include/votca/xtp/mmregion.h

```diff
@@ -44,7 +45,7 @@ class MMRegion : public Region {
     return _segments.end();
   }
 
-  void WritePDB(csg::PDBWriter& writer) const;
+  void WritePDB(csg::PDBWriter<csg::CSG_Topology>& writer) const;
```

Comment:
why the template?

Reply:
We have talked about this before, it was related to the use of the topology object in xtp before you made changes to this object. I had made the file readers and writers generic to handle topology objects from csg, xtp and even ctp if needed. At this point I don't really care to revert the changes. We can talk about it more if it bothers you.